ppp_datamodel/utils.py

```diff
@@ -9,3 +9,33 @@ def predicate(node, childs):
         else:
             return any(childs.values())
     return tree.fold(predicate)
+
+def isincluded(tree1,tree2):
+    """
+        Return True if and only if tree1 is included in tree2.
+    """
+    if isinstance(tree1,Resource):
+        tree1=List([tree1])
+    if isinstance(tree2,Resource):
+        tree2=List([tree2])
+    if type(tree1) != type(tree2):
```

Comment:
I am not sure to understand. I want `isincluded(Missing(),TypedMissing()) == False` and `isincluded(TypedMissing(),Missing()) == False`.
Keep in mind that I want to use this to test the output of question parsing modules. If I want my module to return `TypedMissing()`, then I want the test to fail if it returns `Missing()`.